crates/libcontainer/src/process/container_intermediate_process.rs

```diff
@@ -130,10 +132,11 @@ pub fn container_intermediate_process(
                     }
                     if let ContainerType::TenantContainer { exec_notify_fd } = args.container_type {
                         let buf = format!("{e}");
-                        if let Err(err) = write(exec_notify_fd, buf.as_bytes()) {
+                        let exec_notify_fd = unsafe { std::fs::File::from_raw_fd(exec_notify_fd) };
```

Comment:
why not use `unsafe { std::os::fd::OwnedFd::from_raw_fd( exec_notify_fd ) }`
i think `RawFd` is more semantically related to `OwnedFd` instead of `File`

Reply:
Oh Yes ! That is much better. Will change it.

Reply:
Marking as resolved, as changed accordingly